.github/actions/compile/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
         shell: bash
 
       - name: Run Tests
-        continue-on-error: true
+        continue-on-error: false
```

Comment:
It's also necessary to remove the `|| echo "Tests failed, but continuing` from line 37.